assets/javascripts/feature.js

```diff
@@ -0,0 +1,101 @@
+function newFeature(featureVersion) {
+
+    var currentFeature;
+
+    //Create variable for each tour
+    var example = new Tour({
+        //Enable to save progress local, necessary for multipage traversal
+        storage: window.localStorage,
+        template: changeTemplate(),
+        onShown: function(){return dontShow(), currentFeature = 2; quitTour(currentFeature)}
+    });
+
+    //Add steps to the tour
+    example.addSteps([
+        {
+          element: ".jumbotron",
+          title: "Interested in new features?",
+          content: "Just click next to see what's new in openQA",
+          placement: "bottom",
+          backdrop: true,
+          onNext: function(){return ($('.panel-heading').click())}
+        },
+        {
+          element: "#filter-fullscreen",
+          title: "Full screen mode",
+          content: "Check out the new full screen view option",
+          placement: "top",
+          backdrop: false,
+          onNext: function(){document.location.href = '/tests/'},
+        },
+        {
+          element: "#scheduled_wrapper",
+          title: "Multipage",
+          content: "Example about traversing across pages",
+          placement: "bottom",
+          onPrev: function(){document.location.href = '/'},
+        }
+    ]);
+
+    initTour(featureVersion);
+
+    //Parse html code as string to change the default layout of bootstrap tour
+    function changeTemplate() {
```

Comment:
you better put that HTML in a hidden block with proper indenting.

But I'll merge (and squash) in this stage.